package.json

```diff
@@ -1,101 +1,101 @@
+    "demo:deploy": "npm run build && npm run jsdoc && npm run demo:prebuild-version && npm run demo:prebuild-latest && npm run demo:build && gh-pages -d demo/_site --add --remote upstream",
+    "demo:origin": "npm run build && npm run jsdoc && npm run demo:prebuild-version && npm run demo:prebuild-latest && npm run demo:build && gh-pages -d demo/_site --add --remote origin",
+    "demo:updateVersion": "echo $npm_package_version > demo/_data/version.yml",
+    "demo:setup": "cpx 'node_modules/@egjs/common-demo/**/*' 'demo/' && rm demo/package.json",
+    "release": "node config/release.js",
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "changelog": "node ./config/changelog.js",
+    "prepush": "npm run lint",
+    "commitmsg": "node config/validate-commit-msg.js"
+  },
+  "author": {
+    "name": "NAVER Corp."
+  },
+  "namespace": {
+    "eg": "eg",
+    "Hammer": "Hammer"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/naver/egjs-axes",
+  "dependencies": {
+    "@egjs/agent": "^2.2.1",
+    "@egjs/component": "^2.2.2",
+    "@egjs/hammerjs": "^2.0.15"
+  },
+  "devDependencies": {
+    "@egjs/common-demo": "github:naver/egjs#common-demo",
+    "ts-loader": "^8.0.6",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "chai": "^4.1.2",
+    "chalk": "^2.3.0",
+    "clean-webpack-plugin": "^0.1.17",
+    "coveralls": "^3.0.0",
+    "cpx": "^1.5.0",
+    "egjs-jsdoc-template": "^1.4.4",
+    "fs-extra": "^5.0.0",
+    "gh-pages": "^1.1.0",
+    "hammer-simulator": "0.0.1",
+    "hammerjs-compatible": "^1.1.0",
+    "husky": "^0.14.3",
+    "inject-loader": "^3.0.1",
+    "istanbul-instrumenter-loader": "^3.0.0",
+    "jsdoc": "^3.5.5",
+    "karma": "^2.0.0",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage-istanbul-reporter": "^1.3.3",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sinon": "^1.0.5",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^2.0.9",
+    "lite-fixture": "^1.0.2",
+    "marked": "^0.3.12",
+    "mocha": "^4.1.0",
+    "rollup": "^0.66.1",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-prototype-minify": "^1.0.1",
+    "rollup-plugin-replace": "^2.1.0",
+    "rollup-plugin-typescript": "^1.0.0",
+    "rollup-plugin-typescript2": "^0.17.1",
+    "rollup-plugin-uglify": "^6.0.0",
+    "sinon": "^4.1.4",
+    "string-replace-webpack-plugin": "0.1.3",
+    "sync-exec": "^0.6.2",
+    "tslint": "^5.9.1",
+    "tslint-eslint-rules": "^4.1.1",
+    "tslint-loader": "^3.5.3",
+    "typescript": "^3.9.7",
+    "uglifyjs-webpack-plugin": "^1.1.6",
+    "webpack": "^3.10.0",
+    "webpack-dev-server": "^2.10.1",
+    "webpack-merge": "^4.1.1",
+    "write-file-webpack-plugin": "^4.2.0",
+    "xml2js": "^0.4.19"
+  }
+}
```

Comment:
Please revise the editorconfig settings, I think "trim trailing whitespace" and "insert final newline" can be added to all file extensions.